firmware/arduino/chessboard/uart.ino

```diff
@@ -28,6 +30,20 @@ enum ErrorCode
     MOVEMENT_ERROR = '4'
 };
 
+// Send message to Pi when the chess timer is pressed
+void chess_timer_ISR()
```

Comment:
The capital ISR doesn't follow the style guide, but chess_timer_isr() seems wrong to me for some reason. I can change it if you'd prefer.